spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/SparkSQLProperties.java

```diff
@@ -70,4 +72,18 @@ private SparkSQLProperties() {}
 
   // Controls whether to report locality information to Spark while allocating input partitions
   public static final String LOCALITY = "spark.sql.iceberg.locality.enabled";
+
+  public static final String EXECUTOR_CACHE_ENABLED = "spark.sql.iceberg.executor-cache.enabled";
+  public static final boolean EXECUTOR_CACHE_ENABLED_DEFAULT = true;
+
+  public static final String EXECUTOR_CACHE_TIMEOUT = "spark.sql.iceberg.executor-cache.timeout";
+  public static final Duration EXECUTOR_CACHE_TIMEOUT_DEFAULT = Duration.ofMinutes(10);
+
+  public static final String EXECUTOR_CACHE_MAX_ENTRY_SIZE =
```

Comment:
what is the guidance for max entry size?  Are we trying to prevent one huge entry preventing smaller entries from getting cached, assuming that we may cache fewer total deletes that way?

Reply:
Yep, that's the use case.